index.html

```diff
@@ -2,15 +2,73 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
 </head>
 
+<section class="header-section">
+    <img class="logo" alt="a red-velvet cupcake logo" src="../images/Redvelvet_cupcake_logo.jpg">
+    <div class="header-right">
+        <nav class="hamburger-menu">
+            <!-- <button id="open">☰</button> -->
+                <div class="hamburger-menu-item">HOME</div>
+                <div class="hamburger-menu-item">CAKES</div>
+                <div class="hamburger-menu-item">ORDERING</div>
+                <div class="hamburger-menu-item">LESSONS</div>
+                <div class="hamburger-menu-item">ABOUT US</div>
+        </nav>
+        <p class="slogan">The best cakes in town delivered to your door</p>
+    </div>
+</section>
+
+
 <body>
+    <main>
+        <section class="large-nav">
+            <div class="large-nav-item">HOME</div>
+            <div class="large-nav-item">CAKES</div>
+            <div class="large-nav-item">ORDERING</div>
+            <div class="large-nav-item">LESSONS</div>
+            <div class="large-nav-item">ABOUT US</div>
+        </section>
+        <div class="desktop-hero">
+            <section class="welcome-section">
+                <h1>Welcome</h1>
+                <p class="site-description">At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita distinctio. Nam libero tempore, cum soluta nobis est eligendi optio cumque nihil impedit quo minus id quod maxime placeat facere possimus, omnis voluptas assumenda est, omnis dolor repellendus.</p>
+            </section>
+    
+            <section class="hero-section">
+                <img id="hero-img" alt="Strawberry cheese cake decorated with strawberry on top" src="../images/Strawberry_Cheesecake.jpg">
+            </section>
+        </div>
+
+        <section class="cake-display">
+            <img alt="pretty cupcakes with pastel coloured icing on top" src="../images/pretty_cupcakes.jpg">
+            <img alt="three layered cake beautifully decorated with icing, strawberries and blueberries" src="../images/Pretty_cake_berry_topping.jpg">
+            <img alt="Chocolate cake with Happy Birthday written on it" src="../images/Chocolate_cake.jpg">
+            <img alt="cake decorated with white icing and strawberries, kiwis, raspberries and blackberries on top" src="../images/Cake_with_fruit_topping.jpg">
+        </section>
+
+
+    </main>
+    
+
     <!-- Add your markup here -->
 </body>
 
+<div class="footer-section">
+    <section class="side-footer">
+        <div class="side-footer-item">Gallery</div>
```

Comment:
The margin above and below these footer items is a little big, so they could be reduced